documentation/dev_setup.md

```diff
@@ -1,60 +1,86 @@
 # Development environment setup
-
-To contribute to TRLC you will need to be able to run the
-testsuite. Currently this is only realistic on GNU/Linux, although it
-may be possible to get it to work on other platforms.
-
+ 
+To contribute to TRLC, you will need to be able to run the
+testsuite. Currently, this is only realistic on GNU/Linux and Windows,
+although it may be possible to get it to work on other platforms.
+ 
```

Comment:
```suggestion
To contribute to TRLC, you will need to be able to run the
testsuite. Therefore you need GNU make and GNU findutils.
We do provide setup steps for Windows and Linux. If you run into any problems on macOS, make sure you install the latest GNU make version.
```

Reply:
It is possible on macOS for sure. I think the only hindrance could be an outdated make version